### ISSUE TITLE
User-defined links to procedures

### ISSUE THREAD

**Issue by jacobwilliams** [cmacmackin/ford#37]
I was wondering if we could have the option of allowing the user to specify a link in the comments from one procedure to another.  For example, [here](http://jacobwilliams.github.io/bspline-fortran/proc/db2ink.html) I refer to the ```db2val``` procedure. It would be great if this was a link that I could click to immediately take me there.  RoboDoc does this automatically (so anywhere I have this string, it will be a link to the procedure)...but sometimes this is too aggressive.  It would be nice if I could specify them using something like ```[db2val]``` or ```[bspline_module:db2val]``` and have this be converted to a link.

**Comment by cmacmackin** [comment]
I can certainly look at doing this. As a short-term workaround, what you can do is `[db2val](|url|/proc/db2val.html)`.

**Comment by zbeekman** [comment]
Yeah robodoc is too aggressive, IMO. I’m unsure of what it does if it thinks there are name conflicts and how it resolves the correct reference, if it is even able to do so.

**Comment by cmacmackin** [comment]
This feature has been added to the GitHub version of FORD. The syntax is a bit different from what you proposed. See [the wiki](https://github.com/cmacmackin/ford/wiki/Writing-Documentation#links) for documentation. Hopefully I'll be authoring a new release for PyPI within the week. In the meantime, if anyone wants to test this feature it would be much appreciated, as I didn't try it out in all of its conceivable permutations. I'm confident that it basically works, but there could be some subtle issues which I missed or places where I made a typo.
